server/api/contests.py

```diff
@@ -113,6 +113,20 @@ def validate_contests(contests: List[JSONDict]):
                 f" ({total_votes} votes, {total_allowed_votes} allowed)"
             )
 
+    # Jointly targeted contests must all have the same contest universe and total ballots
+    targeted_contests = [contest for contest in contests if contest["isTargeted"]]
+    if any(
+        set(contest["jurisdictionIds"]) != set(contests[0]["jurisdictionIds"])
+        for contest in targeted_contests[1:]
+    ):
+        raise BadRequest("All targeted contests must have the same jurisdictions.")
+
+    if any(
+        contest["totalBallotsCast"] != contests[0]["totalBallotsCast"]
+        for contest in targeted_contests[1:]
+    ):
```

Comment:
clean, I like it.

Reply:
maybe small nit to, at some point, create constants of these field names.

Reply:
will make a note to go back and do that everywhere at some point